pkg/metrics/chaos-controller-manager.go

```diff
@@ -27,10 +29,9 @@ import (
 	"github.com/chaos-mesh/chaos-mesh/pkg/status"
 )
 
-var log = ctrl.Log.WithName("chaos-controller-manager-metrics-collector")
-
 // ChaosControllerManagerMetricsCollector implements prometheus.Collector interface
 type ChaosControllerManagerMetricsCollector struct {
+	log                 logr.Logger
```

Comment:
please use logger here :)

Reply:
I'm not sure how I use logger here.

Reply:
@Andrewmatilde Any suggestion here?

Reply:
`logger logr.Logger` is OK.

Reply:
> do you mean like below?
> 
> ```go
> logger logr.Logger
> ```

YES!

Reply:
> I'm not sure how I use logger here.
> 
> becaz `NewDefaultZapLogger()` returns logger and error both,

 You cloud append the `logger` as another parameter in the constructor of `ChaosControllerManagerMetricsCollector`, and do not care about how the `logger` comes here.